packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,158 @@
+- **Field:** Represents a UI element, either a static item, such as a label, or user input, such as a selection list. In addition to recording the associated user input, a field object contains state information, such as whether the associated value is required. Options, or lists of choices, are represented by a collection of fields. A field may also contain a form that contains more fields.
+- **Form:** Contains the fields that represent the user action for a remediation.
+- **Configuration:** Contains the settings used by the SDK to connect to an Application Integration in an Okta org.
+- **Client:** Represents the session during the sign-on flow.
+
+
+## Objects and the flow
+
+<div class="common-image-format">
+
+![A diagram showing the relationship between the SDK objects the sign-in flow.](/img/mobile-sdk/mobile-idx-objects-and-flow.png "A diagram that shows the main objects associated with each stepp in the sign-in flow.")
+
+</div>
+
+The main objects associated with each step in the flow are:
+
+| Sign-in step         | Objects                         |
+| :------------------- | :------------------------------ |
+| Initialize SDK       | Configuration <br/> Client      |
+| Request initial step | Client                          |
+| Receive step         | Response                        |
+| Check signed-in      | Response <br/> Remediation      |
+| Gather user input    | Remediation <br/> Authenticator |
+| Send input           | Remediation                     |
+| Done                 | Response                        |
+
+
+## Common parts of the flow
+
+Some parts of the sign-in flow are the same:
+
+- Add the SDK to your app.
+- Configure the SDK.
+- Initialize the client and start the flow.
+- Process the response.
+- Request a token.
+- Sign the user out.
+
+### Add the SDK
+
+<StackSnippet snippet="adddependency" />
+
+### Create and manage configurations
+
+The values for a configuration object are:
+
+| Value         | Description |
+| :------------ | :---------- |
+| Issuer        | The Oauth 2.0 URL for the Okta org, such as `https://oie-123456.oktapreview.com/oauth2/default`. |
+| Client ID     | The ID of the Okta Application Integration from the Okta Admin Console.  |
+| Redirect URI  | A callback URI for launching the mobile app, such as `com.example.oie-123456:/callback`. |
+| Scopes        | A space separated list of the requested access scopes for the app. For more information, see [Scopes in OpenID Connect & OAuth 2.0 API](https://developer.okta.com/docs/reference/api/oidc/#scopes).|
+
+The configuration information in a shipping app is usually static. You can initialize the configuration values directly in the code or by reading them in from a file. During development you may want to provide a way to edit configuration values.
+
+<StackSnippet snippet="loadingaconfiguration" />
+
+### Start sign-in
+
+Start the sign-in flow by creating an SDK client using a configuration, and then requesting the first step.
```

Comment:
I debated that one... In this case I'm referring to a concept, the configuration for the SDK client, rather than to a specific instance of the Configuration object. I'm fine either way.

Reply:
I think it's going to be ambiguous to readers. Could we say "build configuration" here?

Reply:
How about modifying it to "an initialized configuration object" or something similar?

The section above defines what that is, so the context should either be clear (reading through the guide) or easy to locate (the word configuration appears several times.)

Reply:
You could just delete "using a configuration". The code demonstrates this anyway.

Reply:
@danielmaharry-okta , that's a great suggestion. Done.